### DISCUSSION TITLE
今天早上扔垃圾的时时候遇见一位灰色头发的女生在翻垃圾桶。

### DISCUSSION THREAD

**Discussion by Rain-YYC** [share121/inter-knot#2617] [其他]
![289aa83995ec0947627ee08458250a2](https://github.com/user-attachments/assets/76381c79-1d07-46f1-af85-0695b78da1ff)
这位女士我似乎感觉在哪里见过，大家有什么头绪？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Eurus1997** [comment]
那天吃拉面，听到隔壁的人在讨论六分街和光映广场的垃圾桶特别奇怪，经常能看到一个灰毛在那一边翻垃圾桶一边念念有词什么“开拓”“星穹列车”一些听不懂的话，像是精神有什么问题。过一会人又离奇消失不知道去哪里了。我听着有点吓人，不知道要不要去报治安官。

**Comment by Kapler186f** [comment]
嗯呢，嗯呢嗯呢。（不知道，店长没教）

**Comment by yivitte** [comment]
三月七：开拓者！开拓者！丸辣！开拓者失踪了！快把开拓者还我
